products/ha_memgraph/v0.15.0/README.md

```diff
@@ -0,0 +1,32 @@
+# Technical Documentation
+
+## About Memgraph
+
+HA Memgraph is an ACID compliant, highly available, transactional graph
```

Comment:
Should use add `in memory graph database` here?

Reply:
Yes, I agree. The about should be aligned with info about Memgraph.